core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCatalog.java

```diff
@@ -52,19 +53,23 @@
 import org.apache.iceberg.transforms.Transforms;
 import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 public class TestHadoopCatalog extends HadoopTableTestBase {
   private static ImmutableMap<String, String> meta = ImmutableMap.of();
 
-  @Test
-  public void testCreateTableBuilder() throws Exception {
+  @ParameterizedTest
```

Comment:
It did not make sense to parameterize the entire suite, so I only changed applicable tests.